manifests/modeljob-operator/templates/deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
+            - name: KLEVER_MODEL_REGISTRY_ADDRESS
```

Comment:
What's the difference between KLEVER_MODEL_REGISTRY_ADDRESS and  SERVER_ORMB_DOMAIN

Reply:
Notes have instructions
```
// getORMBDomain is get domain for ModelJob task.
// For model extraction, when it is completed, it should push model to harbor directly
// so that not create ModelJob repeatedly, but for model conversion, when convert complete,
// it should push to klever-model-registry, so that it can extract model automatically.
```